code/modules/antagonists/traitor/sleeper_agent.dm

```diff
@@ -5,7 +5,10 @@
 	antagonist_icon = "traitor"
 
 /datum/antagonist/sleeper_agent/announce()
-	boutput(owner.current, "<h3><span class='alert'>You have awakened as a Syndicate [display_name]!</span></h3>")
+	var/mob/living/carbon/human/H = src.owner.current
+	boutput(H, "<h3><span class='alert'>You have awakened as a Syndicate [display_name]!</span></h3>")
+	new /obj/item/implant/access/infinite/sleeper(H)
+	boutput(owner.current, "You feel something activate within you, and a vision of a place hidden in the asteroids near the station.")
```

Comment:
```suggestion
	boutput(H, "You feel something activate within you, and a vision of a place hidden in the asteroids near the station.")
```
`owner.current` got reverted in what I assume was a merge conflict between your origin and local branches. Ensure to pull changes from your origin prior to making changes on your local.

I also feel that the text is now too ambiguous.